### PR TITLE
Add verbose option for resend subscriptions

### PR DESCRIPTION
**Pull request by codyebberson**

### CODE REVIEW

**Review by rahul1 — APPROVED** [review]

File: packages/server/src/fhir/repo.ts [hunk 1]
```diff
@@ -188,9 +188,13 @@ export interface CacheEntry<T extends Resource = Resource> {
   projectId: string;
 }
 
-export type ReadResourceOptions = {
+export interface InteractionObtions {
```
Comment (rahul1) [inline packages/server/src/fhir/repo.ts:191]:
```suggestion
export interface InteractionOptions {
```

Comment (codyebberson) [reply packages/server/src/fhir/repo.ts:191]: Eek, thanks.  pesky `d` `b` `p` `q`

File: packages/server/src/seed.test.ts [hunk 2]
```diff
@@ -39,5 +39,5 @@ describe('Seed', () => {
 
     // Second time, seeder should silently ignore
     await seedDatabase();
-  }, 240000);
+  }, 600000);
```
[the remark this answers is not in the document]
Comment (ThatOneBro) [reply packages/server/src/seed.test.ts:42]: Also prefer to increase the timeouts at the `test:seed` level
https://github.com/medplum/medplum/blob/49e71515c852c5c78da0d0bbfff2710be1b7902b/packages/server/package.json#L22

[the remark this answers is not in the document]
Comment (codyebberson) [reply packages/server/src/seed.test.ts:42]: > Curious why we're increasing the seed test duration so high?

It was timing out on my machine (i9-10850K CPU, 10 Cores, 3.60GHz).  It "only" took 320 seconds, but this is a decently powered machine so I'm worried that others might run into timeouts as well.

Running Postgres on Docker+WSL is probably contributing to the problem.

We need to collapse the database migrations in `v4`, that will make a huge difference.

> Also prefer to increase the timeouts at the test:seed level

K, I can do that.

**Pull request merged**